.github/workflows/manual-docker-release.yml

```diff
@@ -0,0 +1,77 @@
+name: manual-docker-release
+
+env:
+  GPG_KEY_ID_GITHUB_TECHUSER: '${{ secrets.GPG_KEY_ID_GITHUB_TECHUSER }}'
+  PROJECT_NAME: ${GITHUB_REPOSITORY#*/}
+  DOCKER_BAKE_PLATFORM: linux/amd64,linux/arm64
+
+on:
+  push:
+    tags:
+      - dookug-*
+  workflow_run:
+    workflows: [ "manual-maven-release" ]
+    branches: [ "main" ]
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      comment:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: 'Comment for release'
+        # Default value if no value is explicitly provided
+        default: 'Start docker release from WEB UI'
+        # Input has to be provided for the workflow to run
+        required: true
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    # run only on main or tags
+    if: ((github.ref_name == 'main' && contains('["rombow","czenczl","petrenyi-mark","speter555","bucherarnold","biromate","cserhatit"]', github.actor)) || startsWith(github.ref, 'refs/tags/'))
```

Comment:
Ez így tuti oké? Ha nem nézem be a zárójelezést, akkor tag-ről bárki indíthatja?

Reply:
ezt nem is neztem, csak atmasoltam egy masikbol

Reply:
lasd #99